src/components/WeatherCardComponent/styles.js

```diff
@@ -1,14 +1,21 @@
 import styled from "styled-components";
 
 export const WeatherCard = styled.div`
-  width: 1280px;
   height: 720px;
```

Comment:
If you want to use fixed height/width it's better to use 'rem' instead of 'px'

1 rem = 16px 

It's better because of user can have his own browser style settings and with 'rem' it will be more adaptive